src/infra/workload/releaseunit/modules/stamp/policy.tf

```diff
@@ -52,4 +52,24 @@ resource "azurerm_resource_group_policy_assignment" "aks_rbac_enabled" {
   resource_group_id    = azurerm_resource_group.stamp.id
   policy_definition_id = "/providers/Microsoft.Authorization/policyDefinitions/ac4a19c2-fa67-49b4-8ae5-0b2e78c49457"
   display_name         = "Role-Based Access Control (RBAC) should be used on Kubernetes Services"
-}
\ No newline at end of file
+}
+
+# Limit access to external container registries (BuiltIn)
+# https://github.com/Azure/azure-policy/blob/master/built-in-policies/policyDefinitions/Kubernetes/ContainerAllowedImages.json
+resource "azurerm_resource_group_policy_assignment" "aks_limit_registries" {
+  name                 = "aks-limit-registries"
+  resource_group_id    = azurerm_resource_group.stamp.id
+  policy_definition_id = "/providers/Microsoft.Authorization/policyDefinitions/febd0533-8e55-448f-b837-bd0e06f16469"
+  display_name         = "Kubernetes cluster containers should only use allowed images"
```

Comment:
```suggestion
  display_name         = "Kubernetes cluster containers should only be pulled from allowed container registries"
```

Reply:
The name used is the name of the policy. https://github.com/Azure/azure-policy/blob/master/built-in-policies/policyDefinitions/Kubernetes/ContainerAllowedImages.json#L3

Reply:
hm fair point. But what if we find that name misleading? :D I'd assume we can call it here whatever we want?

Reply:
We of course can - and I've added a link to the GitHub repo. But, to make it easier I'd prefer to use the exact same name you'd find in the portal. Even though in this specific case it's not perfect - though its not wrong. You can use it for container image names as well as for registries.

Reply:
Maybe you'd want to raise a PR here: https://github.com/Azure/azure-policy/blob/master/built-in-policies/policyDefinitions/Kubernetes/ContainerAllowedImages.json#L3